modules/gpgsqlbackend/4.3.0_to_4.7.0_schema.pgsql.sql

```diff
@@ -1,13 +1,5 @@
-BEGIN;
-  ALTER TABLE domains ADD COLUMN options VARCHAR(65535) DEFAULT NULL;
-  ALTER TABLE domains ADD COLUMN catalog VARCHAR(255) DEFAULT NULL;
+ALTER TABLE domains ADD COLUMN options VARCHAR(65535) DEFAULT NULL;
+ALTER TABLE domains ADD COLUMN catalog VARCHAR(255) DEFAULT NULL;
+ALTER TABLE domains ALTER COLUMN type VARCHAR(8) NOT NULL;
```

Comment:
`ALTER TABLE domains ALTER COLUMN type VARCHAR(8) NOT NULL;` this is missing the `TYPE` keyword.

I'd suggest instead:

```
ALTER TABLE domains ALTER COLUMN type TYPE text;
```

And:
```
ALTER TABLE domains ADD COLUMN options TEXT DEFAULT NULL,
ADD COLUMN catalog TEXT DEFAULT NULL;
```
(i.e. both new columns in one statement)

My suggestions also use TEXT, as on the postgres side there is no real difference, and in the future you don't need to change the column types again.

Reply:
amended